### DISCUSSION TITLE
How to make ListView scrolling available inside a ScrollView on `Android`?

### DISCUSSION THREAD

**Discussion by dim-37** [unoplatform/uno#14258] [Q&A]
I am encountering an issue on Android application where I have a ScrollView containing multiple elements, including a ListView. Unlike the Windows version, where both the ScrollView and ListView scroll correctly, on Android, the ListView does not respond to scrolling attempts. How can I resolve this issue?

**Comment by jeromelaban** [comment]
Thanks for the question. 

In general, having a ListView inside of a ScrollViewer is a big red flag, as it will cause large performance issues by disabling virtualization. Why do need such a UI structure?

**Reply by xperiandri** [reply]
We need to implement a curtain control with a list of grouped filters inside it. Virtualization is not necessary at all as there are not many filters.
But even if it were important we would just constrain the `ListView` height.

The curtain is a mandatory snap `ScrollViewer` inside a `Popup` that acts similar to iPhone's setting curtain but from bottom to top
<img src="https://user-images.githubusercontent.com/2365592/280495674-2e1fa608-7116-4bb7-b9e3-c0a7cf9fc242.png" alt="image" width="200">

**Reply by xperiandri** [reply]
Depending on content it can be in several states:
* Closed – only `Thumb` is visible
* HeaderOpen – a piece of curtain is visible from bottom
* FullOpen – content occupies almast all screen but a piece of bottom layer is visible in the top
* FillScreen – `Thumb` is hidden under the top of the screen

I have no other idea to implement that except using a scrollviewer with mandatory snap points

**Reply by jeromelaban** [reply]
I see, thanks. You may want to use either an `ItemsControl` or an `ItemsRepeater` instead, if you're not using features that are coming from `ListView` specifically.
